src/utils/scrubber.coffee

```diff
@@ -86,6 +86,13 @@
 
           positions[index] = {index, x: x, y: axes[v.axis + 'Scale'](v.y + v.y0), side, sizes}
 
+          # Use a coloring function if defined, else use a color string value
+          color = series.color?(v, series.values.indexOf(v)) ? series.color
```

Comment:
Could you use a more explicit syntax here ? Like `color = if angular.isFunction(series.color) then series.color(v, series.values.indexOf(v)) else series.color` or something like that ?